src/main/java/io/kuzzle/sdk/API/Controllers/DocumentController.java

```diff
@@ -0,0 +1,53 @@
+package io.kuzzle.sdk.API.Controllers;
+
+import io.kuzzle.sdk.CoreClasses.Maps.KuzzleMap;
+import io.kuzzle.sdk.CoreClasses.Responses.Response;
+import io.kuzzle.sdk.Events.Event;
+import io.kuzzle.sdk.Exceptions.InternalException;
+import io.kuzzle.sdk.Exceptions.NotConnectedException;
+import io.kuzzle.sdk.Kuzzle;
+
+import java.util.ArrayList;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ConcurrentHashMap;
+
+public class DocumentController extends BaseController {
+  public DocumentController(final Kuzzle kuzzle) {
+    super(kuzzle);
+  }
+
+  /**
+   * Creates a document in a given collection and index.
+   *
+   * @param index
+   * @param collection
+   * @param document
+   * @param _id
+   * @param options
+   * @return a CompletableFuture
+   * @throws NotConnectedException
+   * @throws InternalException
+   */
+  public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> document,
+      final String _id,
+      final ConcurrentHashMap<String, Object> options) throws NotConnectedException, InternalException {
+    final KuzzleMap query = new KuzzleMap();
+
+    query
+        .put("index", index)
+        .put("collection", collection)
+        .put("controller", "document")
+        .put("action", "create")
+        .put("_id", _id)
+        .put("body", document)
+        .put("options", options);
```

Comment:
that's not how options should be handled, check the `query` method

Reply:
Obviously, that was a mistake, sorry